pjp.tm10/src/main/webapp/FormattingWithCSS/css/ALL.css

```diff
@@ -0,0 +1,124 @@
+@charset "ISO-8859-1";
+#Q1body
+{
+	background-image: url("../img/img1.png");
+	background-repeat: no-repeat;
+	background-position: right top;	

```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5784138)